lib/smart_proxy_remote_execution_ssh/runners/script_runner.rb

```diff
@@ -340,6 +342,14 @@ def local_command_file(filename)
       File.join(ensure_local_directory(local_command_dir), filename)
     end
 
+    def socket_dir
+      File.join(@socket_working_dir, 'sockets', "foreman-ssh-cmd-#{@id}")
```

Comment:
The `sockets` and `foreman-ssh-cmd-$UUID` subdirectories are not really necessary. The user configures a directory in which the control sockets should live. I'd be ok with putting all the sockets directly into that directory with job id as the name for each of them. I don't think we'll need more than one control socket per job in the foreseeable future